test/rpc_test.py

```diff
@@ -1090,6 +1104,50 @@ def test_call_method_on_rref(self):
 
         self.assertEqual(result, sum(vals))
 
+    @dist_init(setup_rpc=False)
+    @requires_process_group_agent("PROCESS_GROUP rpc backend specific test, skip")
+    def test_rpc_shutdown(self):
+        rpc.init_rpc(
+            name="worker%d" % self.rank,
+            backend=rpc.backend_registry.BackendType[
+                dist_utils.TEST_CONFIG.rpc_backend_name
+            ],
+            rank=self.rank,
+            world_size=self.world_size,
+            init_method=self.init_method,
+            rpc_agent_options=self.rpc_agent_options,
```

Comment:
IIUC, `self. rpc_agent_options` no longer exists, this needs to be replaced by  `rpc_backend_options`?

Reply:
Sorry, looks like the update was coming just as you reviewed